src/cloudinary-image.component.ts

```diff
@@ -104,7 +131,17 @@ export class CloudinaryImage
 
   setElementAttributes(element, attributesLiteral) {
     Object.keys(attributesLiteral).forEach(attrName => {
-      element.setAttribute(attrName, attributesLiteral[attrName]);
+      const attr = attrName === 'src' && this.loading === 'lazy' ? 'data-src' : attrName;
+      element.setAttribute(attr, attributesLiteral[attrName]);
     });
   }
+
+  placeholderHandler(options) {
+    const placeholderOptions = {};
+
+    Object.keys(options).forEach(name => {
+      placeholderOptions[name] = (name === 'width' && !options[name].startsWith('auto') || name === 'height') ? Math.floor(parseInt(options[name], 10) * 0.1) : options[name];
+    })
```

Comment:
@strausr , what does that do?

Reply:
Here, we check for width and height. If we have width and height we change the option to 10% of that. Meaning the actual requested resource is much smaller resulting in a speedy load. This won't change the size of the resource since we set the dimensions of the actual view if the placeholder using the passed width and height- https://github.com/cloudinary/cloudinary_angular/blob/02d879152a0648927a24dbc32392ae1249ece5e4/src/cloudinary-placeholder.component.ts#L12